clarity/evaluator/haspi/eb.py

```diff
@@ -1636,29 +2042,55 @@ def ave_covary2(sigcov, sigMSx, thr):
                 wsync6[k, n] = fsync[5, k]
 
     # Sum the weighted covariance values
-    csum = np.sum(np.sum(weight * sigcov))  # Sum of weighted time-freq tiles
+    csum = np.sum(
+        np.sum(weight * signal_cross_covariance)
+    )  # Sum of weighted time-freq tiles
     wsum = np.sum(np.sum(weight))  # Total number of tiles above thresold
-    fsum = np.zeros(6)
```

Comment:
Let's reorder this so all the fsum lines come first, preceded by a single "Sum of weighted time-freq tiles' comment. and then all the ssum lines.  Wil avoid a lot of redundant comments and be visually easier.